doc/ale.txt

```diff
@@ -33,6 +33,8 @@ CONTENTS                                                         *ale-contents*
       erlc................................|ale-erlang-erlc|
     fortran...............................|ale-fortran-options|
       gcc.................................|ale-fortran-gcc|
+		fusionscript..........................ale-fuse-options
```

Comment:
The formatting isn't right here. Use `||` for linking to the options, and align the entries like the other ones.